web/templates/cve.html

```diff
@@ -149,7 +149,47 @@
     </td>
   </tr>
   {% endif %}
-  {% set keytype = ['vulnerable_configuration_cpe_2_2','impactCVSS','exploitCVSS' ,'cvss', 'capec', 'access', 'impact',
+  <tr>
+    <td>CVSS v3.1</td>
+    <td class="info">
+      <table class="invisiTable">
+        <tr><td><b>Base:          </b></td><td>{{ cve['cvss3'] }} {% if 'cvss-time' in cve %}(as of {{ cve['cvss-time'].strftime('%d-%m-%Y - %H:%M') }}){% endif %}</td></tr>
+      </table>
+    </td>
+  </tr>
+  {% if 'impact3' in cve %}
+  <tr>
+    <td>Impact v3.1</td>
+    <td class="info">
+      <table class="table table-hover table-bordered cve-info">
+        <thead><tr class="thead-dark"><th>Confidentiality</th><th>Integrity</th><th>Availability</hd></tr></thead>
+        <tr>
+          <td class="impact-{{cve['impact3']['confidentiality']|lower}}">{{cve['impact3']['confidentiality']}}</td>
+          <td class="impact-{{cve['impact3']['integrity']|lower}}">{{cve['impact3']['integrity']}}</td>
+          <td class="impact-{{cve['impact3']['availability']|lower}}">{{cve['impact3']['availability']}}</td>
+        </tr>
+      </table>
+    </td>
+  </tr>      
+  {% endif %}
+  {% if 'exploitability3' in cve %}
+  <tr>
+    <td>Exploitability v3.1</td>
+    <td class="info">
+      <table class="table table-hover table-bordered cve-info table-even" style="width:850px">
+        <thead><tr class="thead-dark"><th>Attack Complexity</th><th>Attack vector</th><th>Privileges Required</th><th>Scope</th><th>User Interaction</th></tr></thead>
+        <tr>
+          <td>{{cve['exploitability3']['attackcomplexity']}}</td>
+          <td>{{cve['exploitability3']['attackvector']}}</td>
+          <td>{{cve['exploitability3']['privilegesrequired']}}</td>
+          <td>{{cve['exploitability3']['scope']}}</td>
+          <td>{{cve['exploitability3']['userinteraction']}}</td>
+        </tr>
+      </table>
+    </td>
+  </tr>
+  {% endif %}
```

Comment:
I woul make this entire addition conditional; I would check if cve['cvss3'] is not None and then insert these rows; otherwise just leave them out.